cdbot/cogs/cyber.py

```diff
@@ -360,6 +360,74 @@ def __init__(self, bot: Bot):
         await ctx.send(f"There are {len(game_r.members)} that have completed Cyberstart Game. Out of them, "
                        f"{len(true_r.members)} have also completed Essentials and Assess.")
 
+    @command()
+    async def elitecountdown(self, ctx: Context, event_name: str = None, event_level: str = None):
+        """
+        Gets the time until elite, and shows in an embed
+        """
+        valid_elite_locations = ["London", "Birmingham", "Lancaster "]
+
+        # Validate and normalise inputs
+        event_name = event_name.capitalize()
+        if event_name:
```

Comment:
This line is useless

Reply:
incorrect, `event_name` can be `None`

Reply:
Even if it's None it won't change anything.